modules/client-configuration/pages/contact-methods-saltbundle.adoc

```diff
@@ -27,10 +20,7 @@ The Salt Bundle is provided as the [package]``venv-salt-minion`` that consists o
 Bootstrapping with {webui} is using Salt Bundle as well, so bootstrapping with {webui} is not Python dependant.
 Using the Salt Bundle, it is no longer needed that the client provides any Python interpreter or modules.
 
-Before bootstrapping new client you need to include [package]``venv-salt-minion`` to the bootstrap repository relevant to the client with explicitly specifying [package]``venv-salt-minion`` as an additional package on calling [command]``mgr-create-bootstrap-repo``.
-----
-mgr-create-bootstrap-repo -c DISTRIBUTION_LABEL venv-salt-minion
-----
+If you bootstrap new clients, registration with the Salt Bundle is the default method.
```

Comment:
I would put this as an `[IMPORTANT]` note, to make it notice that the "Salt Bundle has become the default method in 4.2, maybe we should put the new IMPORTANT note at line 15 of this page, where you removed the previous one.

Then here, we can simply reiterate this in a normal sentenace like the this one you added.

Reply:
I added the `[IMPORTANT]` note as suggested.